controllers/datadogagent/controller_test.go

```diff
@@ -353,9 +350,6 @@ func TestReconcileDatadogAgent_Reconcile(t *testing.T) {
 				if err := c.Get(context.TODO(), types.NamespacedName{Name: rbacResourcesName}, clusterRoleBinding); err != nil {
 					return err
 				}
-				if !CheckOwnerReference(datadogAgent, clusterRoleBinding) {
```

Comment:
should we check here instead the Annotation `part-of` ?

Reply:
I pushed a new commit that replaces those owner refs checks with the `part-of` ones 👍